bin/container

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-docker exec -ti phe-cv19-site_web_1 bash
+docker exec -ti ${PWD##*/}_web_1 bash
```

Comment:
what does the `##*/` bit do?

Reply:
ah trims the results of PWD. thats interesting!

Reply:
yeah, just returns the last directory in the path